### PR TITLE
Structure Settings

### PR DESCRIPTION
**Pull request by MohammadKoolivand**
structure of project like files and folders has been created

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PORT = 8000
+DATA_BASE = taskmanager
```
Comment (farshid0594) [inline .env]: delete this file from git

File: src/controllers/admin/tes.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+test
```
Comment (farshid0594) [inline src/controllers/admin/tes.txt]: create readme file instead of tes.txt

**Review by farshid0594 — CHANGES_REQUESTED** [review]
create readme file instead of tes.txt

**Review by farshid0594 — CHANGES_REQUESTED** [review]
change file name to README.md

**Review by farshid0594 — CHANGES_REQUESTED** [review]
remove .env file and push

**Review by amirhnr — APPROVED** [review]

**Review by farshid0594 — APPROVED** [review]

**Pull request merged**